### DISCUSSION TITLE
Feature request:  student name display

### DISCUSSION THREAD

**Discussion by tbflowers** [openwebwork/webwork2#2217] [Ideas]
When acting as a student on a problem page, there is a drop down with the student's name displayed. It stays there in the top right as you scroll down the page. 

Could there be an option in the Configuration menus to turn on that name display in the same place for a student? In other words, a student would see their own name displayed there on each problem. This would be a nice feature when students are working in WW in a computer classroom or computer lab setting. The instructor/tutor/TA could see the student's name displayed there as they walk around the room.

**Comment by drgrice1** [comment]
This is essentially already done in ww 2.18. The students name is shown in the page header by the logout button, and the page header starts there as you scroll down.
